packages/story-editor/src/components/audioPlayer/index.js

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import {
+  useState,
+  useEffect,
+  useCallback,
+  useRef,
+} from '@web-stories-wp/react';
+import {
+  THEME_CONSTANTS,
+  themeHelpers,
+  Icons,
+  Button,
+  BUTTON_SIZES,
+  BUTTON_TYPES,
+  BUTTON_VARIANTS,
+} from '@web-stories-wp/design-system';
+import { __ } from '@web-stories-wp/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Tooltip from '../tooltip';
+
+const StyledButton = styled(Button)`
+  ${({ theme }) =>
+    themeHelpers.focusableOutlineCSS(
+      theme.colors.border.focus,
+      theme.colors.bg.secondary
+    )};
+`;
+
+const Wrapper = styled.div(
+  ({ theme }) => css`
+    position: relative;
+    min-width: 40px;
+    width: 100%;
+    height: 36px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    color: ${theme.colors.fg.primary};
+
+    ${themeHelpers.expandPresetStyles({
+      preset: {
+        ...theme.typography.presets.paragraph[
+          THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL
+        ],
+      },
+      theme,
+    })};
+  `
+);
+
+function AudioPlayer({ title, src }) {
+  const [isPlaying, setIsPlaying] = useState(false);
+
+  const playerRef = useRef();
+
+  const startPlayer = useCallback(() => setIsPlaying(true), [setIsPlaying]);
+  const stopPlayer = useCallback(() => setIsPlaying(false), [setIsPlaying]);
+
+  useEffect(() => {
+    const player = playerRef.current;
+
+    if (!player) {
+      return;
+    }
+
+    if (isPlaying) {
+      player.play();
+    } else {
+      player.pause();
+    }
+  }, [isPlaying]);
+
+  return (
+    <Wrapper>
+      {/* eslint-disable-next-line jsx-a11y/media-has-caption */}
+      <audio crossOrigin="anonymous" loop ref={playerRef}>
+        <source src={src} />
```

Comment:
Can we add mime type here?

Reply:
Hmm the mime type is not actually saved together with the audio, just the URL. So the mime type information is not available.

It's [optional](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/source) anyway though.

I'll check how feasible it would be to add this though.

Reply:
I think we should save some other data here. For resources, we save, mimeType, id and url. This data is extremely useful. I think we should try and save id, as if need to get more data, like sound length or data created, we could get this from making the a call the rest api call to the media endpoint. 

Thoughts?

Reply:
Good example to copy here would be how we handle tracks. 

https://github.com/google/web-stories-wp/blob/17bb7e55d8b17af0e01fa22d2d84ee8e0d9db0f5/packages/story-editor/src/components/panels/design/captions/captions.js#L127-L135

Side note, we should have a prop type for track data and it is a structured object.

Reply:
Addressed now